src/filters/index.js

```diff
@@ -26,6 +28,8 @@ export default {
   exists: existsFilter,
   exist: existsFilter,
   fuzzy: fuzzyFilter,
+  geo_bounding_box: geoBoundingBoxFilter,
+  geo_distance: geoDistanceFilter,
```

Comment:
Could also add `geo-distance` and `geoDistance` as other matching strings for this filter. The goal is to make sure whatever the user guessed is likely to return the filter that they expected.